modules/10-basics/25-evaluation-order/description.en.yml

```diff
@@ -1,26 +1,26 @@
 ---
 
-name: Порядок вычисления
+name: Evaluation order
 theory: |
-  Попробуем превратить выражение `5 - 3 + 1` в программу на Clojure. С точки зрения арифметики, порядок вычисления элементов этого составного выражения строго определен. Сначала вычисляется `5 - 3`, затем к получившемуся результату прибавляется единица.
+  Let's try to turn the expression `5 - 3 + 1` into a Clojure program. From the point of view of arithmetic, the order of calculating the elements of this compound expression is strictly defined. First, `5 - 3` is calculated, then one is added to the result.
 
-  Начнем с того, что вычисляется первым: `5 - 3` превращается в `(- 5  3)`. Затем сложим получившийся результат с единицей: `(+ (- 5 3) 1)`. Так как сложение — это коммутативная операция (помните "от перемены мест слагаемых сумма не меняется"?), то же самое можно записать и в другом порядке: `(+ 1 (- 5 3))`. Неизменным остается то, что в начале каждого списка находится операция.
+  Let's start with what is calculated first: `5 - 3` turns into `(- 5 3)`. Then we add the result to one: `(+ (- 5 3) 1)`. Since addition is a commutative operation (remember "the sum does not change by changing the places of the summands"?), the same can be written in another order: `(+ 1 (- 5 3))`. What does not change is that there is an operation at the beginning of each list.
 
-  Заметьте, выражения, вычисляемые первыми, находятся в глубине дерева. Такое поведение свойственно большинству обычных языков. Сначала вычисляются аргументы функций, затем вызывается сама функция.
+  Notice that the expressions evaluated first are at the bottom of the tree. This behavior is common to most conventional languages. Function arguments are evaluated first, then the function itself is called.
 
-  Попробуем другой вариант: `5 - (3 + 1)`. В этом выражении скобки устанавливают другой приоритет. Это значит, что сначала вычислится сумма единицы и тройки. Так и запишем `(+ 3 1)` (или так `(+ 1 3)`). Теперь возьмём пятерку и вычтем из неё получившийся результат: `(- 5 (+ 1 3))`.
+  Let's try another version: `5 - (3 + 1)`. In this expression, the parentheses set a different priority. It means that the sum of one and three will be calculated first. So we write `(+ 3 1)` (or so `(+ 1 3)`). Now let's take the five and subtract the result from it: `(- 5 (+ 1 3))`.
```

Comment:
the parentheses set a different priority => the parentheses set a different **precedence**